app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/HealthCheckServiceCEImpl.java

```diff
@@ -30,11 +29,10 @@ public HealthCheckServiceCEImpl(
 
     @Override
     public Mono<String> getHealth() {
-        return Mono.zip(getRedisHealth(), getMongoHealth()).map(tuple -> "All systems are Up");
+        return Mono.when(getRedisHealth(), getMongoHealth()).thenReturn("All systems are Up");
```

Comment:
Only because you are modifying this line already. 😄 

And thanks for this imrpovement!

```suggestion
        return Mono.when(getRedisHealth(), getMongoHealth()).thenReturn("All systems are up");
```